src/data/queries/pressRelease.ts

```diff
@@ -0,0 +1,59 @@
+import { QueryData, QueryFormatter, QueryParams } from 'next-drupal-query'
+import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
+import { NodePressRelease } from '@/types/drupal/node'
+import { PressRelease } from '@/types/formatted/pressRelease'
+import { RESOURCE_TYPES } from '@/lib/constants/resourceTypes'
+import { ExpandedStaticPropsContext } from '@/lib/drupal/staticProps'
+import {
+  entityBaseFields,
+  fetchSingleEntityOrPreview,
+} from '@/lib/drupal/query'
+import { get } from 'lodash'
+import { getNestedIncludes } from '@/lib/utils/queries'
+
+// Define the query params for fetching node--press_release.
+export const params: QueryParams<null> = () => {
+  return new DrupalJsonApiParams().addInclude([
+    ...getNestedIncludes('field_press_release_downloads', ["media--image","media--document", "media--video"] ),
+    'field_press_release_contacts',
+    'field_press_release_pdf_version',
+    'field_press_release_office',
+     ])
+}
+
+// Define the option types for the data loader.
+export type PressReleaseDataOpts = {
+  id: string
+  context?: ExpandedStaticPropsContext
+}
+
+// Implement the data loader.
+export const data: QueryData<PressReleaseDataOpts, NodePressRelease> = async (
+  opts
+): Promise<NodePressRelease> => {
+const entity = (await fetchSingleEntityOrPreview(
+    opts,
+    RESOURCE_TYPES.PRESS_RELEASE,
+    params
+  )) as NodePressRelease
+
+  return entity
+}
+
+export const formatter: QueryFormatter<NodePressRelease, PressRelease> = (
+  entity: NodePressRelease
+) => {
+  return {
+    ...entityBaseFields(entity),
+    title: entity.title,
```

Comment:
`title` is include in `entityBaseFields`, so it can be removed.

drupal fields are snake_case, so all of the values coming it will need updated. something like:
```js
  return {
    ...entityBaseFields(entity),
    releaseDate: entity.field_release_date,
    pdfVersion: entity.field_pdf_version.field_document.uri.url,
    introText: entity.field_intro_text,
    address: entity.field_address,
    fullText: entity.field_press_release_fulltext.processed,
    contacts: entity.field_press_release_contact,
    downloads: entity.field_press_release_downloads,
    listing: entity.field_listing?.path.alias,
    administration: {
      id: entity.field_administration?.drupal_internal__tid || null,
      name: entity.field_administration?.name || null,
    },
  }
```

I've added the administration from the above include.